packages/core/src/lib/BannerStrip/BannerStrip.styles.ts

```diff
@@ -31,26 +31,33 @@ const getColor = (type?: TBannerStripTypes) => {
 
 type TSectionStyledProps = Pick<
     IBannerStripProps,
-    'height' | 'width' | 'borderRadius' | 'type' | 'position' | 'bgColor'
+    | 'height'
+    | 'width'
+    | 'borderRadius'
+    | 'type'
+    | 'position'
+    | 'bgColor'
+    | 'customize'
 >;
 
 const SectionStyled = styled.section<TSectionStyledProps>`
     ${resetCSS};
```

Comment:
TCustomize also has other style options like `border`, `margin`.. could you please add those too. border default value can be `none` and for margin it can be `0px`

Reply:
Should I also use onHover here? It seems obvious to me that this property is only for buttons but I prefer to ask anyway.

Reply:
yup we can ignore `onHover`, not sure how it can provide any value here